packages/spark-core/components/_buttons.scss

```diff
@@ -21,42 +22,46 @@
 
 .sprk-c-Button--secondary {
   background-color: transparent;
-  border: 2px solid $blue-medium;
-  color: $blue-medium;
+  color: $black;
+  text-transform: uppercase;
 
   &:active,
   &:focus,
   &:hover {
-    background-color: $blue-medium;
-    border: 2px solid $blue-medium;
-    color: $white;
+    background-color: transparent;
+    color: $black;
   }
-}
 
-.sprk-c-Button--tertiary {
-  background-color: $blue-medium;
+  &:hover::after {
+    animation: width-zero-to-full $btn-transition-speed ease-in-out;
+  }
 
-  &:active,
-  &:focus,
-  &:hover {
-    background-color: $blue-dark;
+  &::after {
+    content: '';
+    border-bottom: $btn-border-width solid $red-real;
+    display: block;
+    width: 100%;
+    padding-top: $space-xs;
+    margin: 0 auto;
   }
 }
 
-.sprk-c-Button--removal {
+.sprk-c-Button--tertiary {
   background-color: transparent;
-  border: 2px solid $red-medium;
-  color: $red-medium;
+  border: $btn-border-width solid $black;
+  color: $black;
+  padding: ($space-m - $btn-border-width) $space-l;
```

Comment:
Because tertiary buttons have a border, we have to account for that space to ensure they are the same height as regular buttons.